spark/v3.2/build.gradle

```diff
@@ -60,11 +56,11 @@ project(":iceberg-spark:iceberg-spark-${sparkMajorVersion}_${scalaVersion}") {
     implementation project(':iceberg-orc')
     implementation project(':iceberg-parquet')
     implementation project(':iceberg-arrow')
-    implementation("org.scala-lang.modules:scala-collection-compat_${scalaVersion}")
+    implementation "org.scala-lang.modules:scala-collection-compat_${scalaVersion}:2.6.0"
```

Comment:
why do we need the `2.6.0` now?

Reply:
or should this be using `scalaCollectionCompat`?

Reply:
> why do we need the 2.6.0 now? 

I actually was present before, you can find version declaration in versions.props in master

> or should this be using scalaCollectionCompat?

because it's dynamic and uses scalaVersion we can't just replace it with libs.scala.collection.compat